benches/multi_party_ecdsa/gg18/keygen.rs

```diff
@@ -65,11 +72,11 @@ mod bench {
                 (0..n)
                     .map(|j| {
                         let vec_j = &secret_shares_vec[j];
-                        vec_j[i]
+                        vec_j[i].clone()
                     })
-                    .collect::<Vec<FE>>()
+                    .collect::<Vec<Scalar<Secp256k1>>>()
```

Comment:
I suggest defining `pub type FE = Scalar<Secp256k1>;` at the top of this file. In this way, when we are about to use the same code for Ed25519, we will need to change just one place instead of many. The same is requested for `pub type GE = Point<Secp256k1>;` please!

Reply:
Probably it's multi-party-eddsa who needs to use `Scalar<Secp256k1>` notation? 🙂 `FE` and `GE` come from older version of `curv` lib

Reply:
The next possible step of multi-party-ecdsa evolution is to make it generic over choice of curve. Ie. points and scalars gonna look like `Point<E>` and `Scalar<E>`, and it will work with any curve, not only secp256k1

Reply:
> Probably it's multi-party-eddsa who needs to use Scalar<Secp256k1> notation? 🙂

😊 Yes, exactly it is!

> The next possible step of multi-party-ecdsa evolution is to make it generic over choice of curve...

OK, that would be great! But just out of curiosity, will the ECDSA over a curve other than secp256k1 be called still "ECDSA"? How to distinguish them from each other?

Reply:
Sure, ECDSA is not limited to secp256k1

Reply:
> How to distinguish them from each other?

 At application layer. If you have signature on bitcoin transaction, it's secp256k1 ecdsa. If you got signature in TLS handshake, it's probably secp256r1

Reply:
Thank you for your response! 🙏🏻